src/state/models/feeds/posts.ts

```diff
@@ -25,6 +25,17 @@ import {MergeFeedAPI} from 'lib/api/feed/merge'
 
 const PAGE_SIZE = 30
 
+type FeedType = 'home' | 'following' | 'author' | 'custom' | 'likes' | 'list'
+
+export enum KnownError {
+  FeedgenDoesNotExist,
+  FeedgenMisconfigured,
+  FeedgenBadResponse,
+  FeedgenOffline,
+  FeedgenUnknown,
+  Unknown,
+}
```

Comment:
Not a hill I'd die on, but I think we should only use string enums. These get evaluated to numbers, which makes rearranging difficult, etc.

```
export enum KnownError {
  FeedgenDoesNotExist = 'FeedgenDoesNotExist'
  ...
}
```

Works 100% of the time, very clear.